spec/support/pdf_spec_helper.rb

```diff
@@ -13,4 +13,14 @@ def non_preparer_fields(filepath)
       key.exclude?("intake_specialist_")
     end
   end
+
+  # Only works for Field Type: Button
+  def check_pdf_option(file_path, field_name, value)
+    data_dump = PdfForms.new.call_pdftk(file_path, :dump_data_fields)
+    matching_field = data_dump.split("---").find { |data_field| data_field.match(field_name) }
+    value_pairs = matching_field.split("\n")
+    field_state_options = value_pairs.filter { |pair| pair.match("FieldStateOption") }
+    clean_field_state_options = field_state_options.map { |field| field.sub("FieldStateOption: ", "") }
+    clean_field_state_options.include?(value)
+  end
```

Comment:
wow so sorry but i'm feeling totally incapable of comprehending this haha 😅 would love to tuple about it!

Reply:
tupled in person!